python/samples/04.ai.f.dataSource.azureOpenAI/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Auto generated during deployment
+requirements.txt
+
+.env
+
+# Teams Toolkit
+env/.env.*.user
```

Comment:
Could you please force commit the `/env` folder (after removing your keys)?

Reply:
Done.